sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_sender.py

```diff
@@ -291,14 +291,19 @@ def from_connection_string(
         return cls(**constructor_args)
 
     def send(self, message):
-        # type: (Union[Message, BatchMessage]) -> None
+        # type: (Union[Message, BatchMessage, List[Message]]) -> None
         """Sends message and blocks until acknowledgement is received or operation times out.
 
+        If a list of messages was provided, attempts to send them as a single batch, throwing a
+        `ValueError` if they cannot fit in a single batch.
+
         :param message: The ServiceBus message to be sent.
         :type message: ~azure.servicebus.Message or ~azure.servicebus.BatchMessage
         :rtype: None
-        :raises: ~azure.servicebus.common.errors.MessageSendFailed if the message fails to
-         send or ~azure.servicebus.common.errors.OperationTimeoutError if sending times out.
+        :raises: :class: ~azure.servicebus.common.errors.MessageSendFailed if the message fails to
+         send
+                :class: ~azure.servicebus.common.errors.OperationTimeoutError if sending times out.
+                :class: `ValueError` if list of messages is provided and cannot fit in a batch.
```

Comment:
we have moved exception into `azure.servicebus.exceptions`.
Can you also check and update other places?

Reply:
Adjusted in many locations.